app/Resources/views/committee/timeline/_message.html.twig

```diff
@@ -1,5 +1,5 @@
 {% if is_granted('ROLE_ADHERENT') %}
-    <div class="text--body">{{ item.content|raw }}</div>
+    <div class="committee-event-description text--body">{{ item.content|raw }}</div>
```

Comment:
I don't know about the words themselves (I'll let @marineb decide on that) but you should be careful to use BEM (even if we didn't everywhere, that's possible): http://getbem.com/introduction/